go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/ulikunitz/xz v0.5.10 // indirect
 	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 // indirect
 	golang.org/x/net v0.12.0 // indirect
+
```

Comment:
This probably shouldn't be here. Can you run the following:
```
go mod tidy && go mod vendor
```

Reply:
Fixed